docs/csharp/whats-new/csharp-10.md

```diff
@@ -120,7 +120,7 @@ Prior to C# 10, there were many scenarios where definite assignment and null-sta
 
 ```csharp
 string representation = "N/A";
-if ((c != null) && c.GetDependentValue(out object obj)) == true)
+if ((c != null) && c.GetDependentValue(out object obj) == true)
```

Comment:
The example in the spec is similar to the following:

```suggestion
if ((c != null && c.GetDependentValue(out object obj)) == true)
```

If this wasn't tested in C# 10 vs previous versions, it would be safer to align with the spec example.